change-notes/1.22/analysis-cpp.md

```diff
@@ -13,6 +13,7 @@
 |----------------------------|------------------------|------------------------------------------------------------------|
 | Expression has no effect (`cpp/useless-expression`) | Fewer false positive results | Calls to functions with the `weak` attribute are no longer considered to be side effect free, because they could be overridden with a different implementation at link time. |
 | Suspicious pointer scaling (`cpp/suspicious-pointer-scaling`) | Lower precision | The precision of this query has been reduced to "medium". This coding pattern is used intentionally and safely in a number of real-world projects. Results are no longer displayed on LGTM unless you choose to display them. |
+| Non-constant format string (`cpp/non-constant-format`) | Higher precision | Rewritten using the taint-tracking library; should also improve performance in batched runs. |
```

Comment:
Please change "Higher precision" to "Fewer false positive results" for consistency.

Somebody (me?) has written "Lower precision" above to mean "Lower `@precision`", which is something very different. I'm open to changing "Lower precision" to something less ambiguous.